staff/web/vhost/get_new_hashes.py

```diff
@@ -0,0 +1,9 @@
+import hashlib
+import os
+
+dir = 'images_2-18-2018/'
+#Images from ocfweb/ocfweb/static/img/hosting-logos as of 2/18/18
+
+for f in os.listdir(dir):
+    with open(dir + f, 'rb') as fp:
+        print('#', f, "\n'" + hashlib.md5(fp.read()).hexdigest() + "',")
```

Comment:
Can we just calculate these in the script itself rather than run this and check in the hashes by hand? It will take approximately 0 seconds to calculate the hashes anyway.

Reply:
Is it too slow or something? I'd expect it to take virtually no time at all to hash a dozen tiny images. That would be preferable to hard-coding hashes imo.